### ISSUE TITLE
Install - Full Database - Out of Date

### ISSUE THREAD

**Issue by ghost** [classicdb/database#681]
Can't use InstallFullDB.sh due to ClassicDB_1_4_1_z2525.sql being out of date. Any chance for an update or explain how I can manually setup a fresh database from scratch?

**Comment by cala** [comment]
I used the install script yesterday and everything went fine. The only "bug" I'm aware of is solved here: http://cmangos.net/thread-6744-post-42630.html#pid42630
Can you post the error message you got, please?

**Comment by ghost** [comment]
![classicdb-install-error](https://cloud.githubusercontent.com/assets/8750744/5438181/f5175142-843b-11e4-955e-4a2096d22ea0.png)

**Comment by ghost** [comment]
When I manually drop the db_version table and rerun the InstallFullDB.sh It works up until it applies mango's core updates and I get another db_version error.
![classicdb-install-error2](https://cloud.githubusercontent.com/assets/8750744/5438303/e4be8df0-843c-11e4-8b37-4e5cd2eb40e9.png)

**Comment by ghost** [comment]
Least to say I finally got through the database install (#thestruggle).. But basic stuff like DUPLICATE id blah blah makes installing the DB a day job, because you can't just relaunch the InstallFullDB.sh until you wipe your database and start from scratch again. Would be nice if it continued where it failed last time, or an option to skip the update. Anyway.. so I used the drop table for db_version to get the ClassicDB_1_4_1_z2525.sql to install. As for mango's core updates, I just manually did it since it was only the bottom 3 it was trying to push through. Then I removed/disabled the full db install and mango's core and classicdb updates from the installer since they have already be done and just make the installer fail over and over on retry. Other than that SD2 and ACID installed just fine.

**Comment by Schmoozerd** [comment]
I think i see what is going on:
https://github.com/classicdb/database/blob/classic/InstallFullDB.sh#L132
https://github.com/classicdb/database/commit/da98d424147a1376f4471d2c5707e474b652d556
(and similar commits)

@cala when you use coreupdates in your update/ folder you need to move the marker in the installer script as well.

**Comment by cala** [comment]
Yes, that what I understood when reading the second part of this issue report.
There are a few cleanups pending on Classic DB repository that I intend to do for the holidays season. :smile:

**Comment by cala** [comment]
To summarize @trulynullified : you don't need to install core updates currently, they are already done in Classic DB. Same for SD2 updates.
The only things you need to do at the moment:
- remove the `db_version` table before running the `InstallFullDB.sh` script (this should be fixed soon)
- applied last ACID release

**Comment by Tobschinski** [comment]
I think there are existing errors in our update folder, or updates, that can easily bug out if you apply them twice. The script at least, does not apply every single update, like expected.

Let's see what I can find there...
